### DISCUSSION TITLE
re-opening pull request

### DISCUSSION THREAD

**Discussion by Spongman** [community/community#22449] [New to GitHub]
<p>how do i re-open a pull request? it looks like i’m in some kind of limbo. it’s complaining about unmerged commits, but i cannot re-open the PR. all i can do is delete the branch.</p>
<p>it doesn’t make any sense.</p>
<p><a href="https://github.com/processing/p5.js/pull/3125" rel="nofollow noopener">https://github.com/processing/p5.js/pull/3125</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/58a7fa27765c7e1fa26a563964acf2de906a74d8.png" data-download-href="/uploads/short-url/cEhUdNoRDuPtpCZ8m60AiHcR1aM.png?dl=1" title="Untitled.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/1088194/181070983-bac19ede-32d4-4271-9280-f1f6a13317ee.png" alt="Untitled.png" data-base62-sha1="cEhUdNoRDuPtpCZ8m60AiHcR1aM" width="690" height="95" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/58a7fa27765c7e1fa26a563964acf2de906a74d8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Untitled.png</span><span class="informations">774×107 6.98 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by pederjohnsen** [comment]
<p>I wonder if it’s permission related or some other repository specific setting since it’s not a repository owned by you.</p>

**Comment by mpboom** [comment, marked as answer]
<p>You can re-open it using a button below the comment box.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/7/7aa6a469ac0b53b9b762ba18b897b10adfb573d5.png" data-download-href="/uploads/short-url/hv1hpaEdunZ8KPwK6zlCSyor9nT.png?dl=1" title="Knipsel.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/19948032/181070987-f74dafae-ccaa-4c48-a7bd-5753c95b961d.png" alt="Knipsel.PNG" data-base62-sha1="hv1hpaEdunZ8KPwK6zlCSyor9nT" width="690" height="223" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/7/7aa6a469ac0b53b9b762ba18b897b10adfb573d5_2_690x223.png, https://aws1.discourse-cdn.com/github/original/2X/7/7aa6a469ac0b53b9b762ba18b897b10adfb573d5.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/7/7aa6a469ac0b53b9b762ba18b897b10adfb573d5.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/7/7aa6a469ac0b53b9b762ba18b897b10adfb573d5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Knipsel.PNG</span><span class="informations">798×258 20.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Reply by axelson** [reply]
This was really difficult for me to find today. I was expecting it near the delete branch button as well.

**Reply by shook415p** [reply]
Can I get my money

**Reply by shook415p** [reply]
Reopen pull request

**Comment by alfattal** [comment]
<p>What if the button was dimmed (deactiveated)?</p>

**Comment by ThinkDigitalSoftware** [comment]
<p>There’s no reason? I see it deactivated when the branch has been deleted</p>
